tests/js/spec/views/issueList/actions.spec.jsx

```diff
@@ -23,7 +23,7 @@ describe('IssueListActions', function() {
         });
 
         SelectedGroupStore.records = {};
-        SelectedGroupStore.add([1, 2, 3]);
+        SelectedGroupStore.add(['1', '2', '3']);
```

Comment:
Can we add a test that checks if bulk assigns calls the API endpoint with the correct requests?

Reply:
Added in `actions.spec.jsx`